docs/developers_guide/quick_start.rst

```diff
@@ -44,6 +44,14 @@ you're working on.
 The ``conda`` directory in the repository has a tool ``configure_compass_env.py``
 that can get you started.
 
+You will need to run ``./conda/configure_compass_env.py`` each time you check
+out a new branch or create a new worktree with ``git``.  Typically, you will
+*not* need to run this command when you make changes to files within the
+``compass`` python package.  These will automatically be recognized because
+``compass`` is installed into the conda environment in "editable" mode.  You
+*will* need to run the command if you add new code or data files to the
```

Comment:
Wow, that's good to know. I would change this to "new code files or data files" to be clear, as my experience with compilers led me to think that any code change required a rerun of `./conda/configure_compass_env.py`

Reply:
There aren't any compilers in python, and the "editable" install is a handy way not to need to reinstall the code after every change.  I didn't know about that feature until relatively recently (about a year ago).

I made the change you requested ("code or data files" --> "code files or data files").  I agree, that's clearer.